libraries/3rdparty/lwip/src/portable/arch/unit-test/tap.h

```diff
@@ -0,0 +1,46 @@
+/* minimal TAP: test anything protocol */
+/* https://testanything.org/ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+int _plan;      /* number of tests */
+int _ok_cnt;    /* number of passing tests */
+int _index;     /* ordinal of current test */
+int _verbose;   /* flag: show all expressions */
+
+/* how many tests should run? */
+/* set verbose flag if environment var VERBOSE exists */
+#define plan(x)  do {                               \
+    _plan = (x);                                    \
+    if (_plan)                                      \
```

Comment:
I'd prefer if these `if`,`else` blocks had `{`,`}` around the statements.  Same comment at 32-36 and 41/42.